tests/ScrollSyncNode.spec.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { mount } from "enzyme";
+import { ScrollSync, ScrollSyncNode } from "../src";
+
+describe("ScrollSyncNode", () => {
+  it("should support using ref on ScrollSyncNode", () => {
+    const ref = jest.fn();
+    mount(
+      <ScrollSync>
+        <ScrollSyncNode ref={ref}>
+          <div />
+        </ScrollSyncNode>
+      </ScrollSync>,
+    );
+    expect(ref).toHaveBeenCalledTimes(1);
+    expect(ref.mock.calls[0][0].toString()).toEqual("[object HTMLDivElement]");
+  });
+
+  it("should support using ref on the direct child node", () => {
+    const ref = jest.fn();
+    mount(
+      <ScrollSync>
+        <ScrollSyncNode>
+          <div ref={ref} />
+        </ScrollSyncNode>
+      </ScrollSync>,
+    );
+    expect(ref).toHaveBeenCalledTimes(1);
+    expect(ref.mock.calls[0][0].toString()).toEqual("[object HTMLDivElement]");
+  });
+
+  it("should only call ref on ScrollSyncNode when having duplicate refs", () => {
+    const ref1 = jest.fn();
+    const ref2 = jest.fn();
+    mount(
+      <ScrollSync>
+        <ScrollSyncNode ref={ref1}>
+          <div ref={ref2} />
+        </ScrollSyncNode>
+      </ScrollSync>,
+    );
+    expect(ref1).toHaveBeenCalledTimes(1);
+    expect(ref1.mock.calls[0][0].toString()).toEqual("[object HTMLDivElement]");
+    expect(ref2).toHaveBeenCalledTimes(0);
```

Comment:
I made it so, that if you have double refs, then it will use the top level one.

Reply:
Can we add a `console.warn` and we'll use the top level one?

Reply:
Sure, I can add that.

Reply:
@AhmadMHawwash I added a `console.warn` and updated the tests in e0c44014e3c80ba5e9b02a3ecc19fdef3bda5754

Reply:
Let me know if you want the warning text to be changed.